### PR TITLE
Added new unit test projects to ReactWindows-Universal solution.

### PR DESCRIPTION
**Pull request by vmoroz**
Fixed the Microsoft.ReactNative.Cxx.UnitTests project to not use local files and added the test projects to  ReactWindows-Universal solution to enable verification of the new code for the ReactNative Windows.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/3775)

### CODE REVIEW

File: vnext/ReactWindows-Universal.sln [hunk 1]
```diff
@@ -89,153 +89,219 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Microsoft.ReactNative.Cxx",
...
 		Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems*{da8b35b3-da00-4b02-bde6-6a397b3fd46b}*SharedItemsImports = 9
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|ARM = Debug|ARM
+		Debug|ARM64 = Debug|ARM64
 		Debug|x64 = Debug|x64
 		Debug|x86 = Debug|x86
-		Debug|ARM = Debug|ARM
+		Release|ARM = Release|ARM
+		Release|ARM64 = Release|ARM64
```
Comment (licanhua) [inline vnext/ReactWindows-Universal.sln]: >ARM64 [](start = 10, length = 5)

remove ARM64. we don't support it

Comment (kmelmon) [reply vnext/ReactWindows-Universal.sln]: We do need to support arm64 (neo is arm64), and we have #3629 tracking adding arm64 build flavors to everything.  why do you say we don't support it?

Comment (vmoroz) [reply vnext/ReactWindows-Universal.sln]: The ARM64 was automatically added because the SampleApp solution has it and because I am adding two projects from the SampleApp.sln it also adds the ARM64. I would prefer to keep it if we plan to support it in future, or otherwise we must remove it from the sample projects too.

File: vnext/Microsoft.ReactNative/pch.h [hunk 2]
```diff
@@ -1,4 +1,3 @@
-#pragma once
```
Comment (NikoAri) [inline vnext/Microsoft.ReactNative/pch.h]: >pragma [](start = 1, length = 6)

why is this removed?

File: vnext/Microsoft.ReactNative.Cxx.UnitTests/Microsoft.ReactNative.Cxx.UnitTests.vcxproj [hunk 3]
```diff
@@ -8,11 +8,13 @@
     <VCProjectVersion>15.0</VCProjectVersion>
     <ProjectGuid>{6c60e295-c8ca-4dc5-b8be-09888f58b249}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
-    <RootNamespace>Microsoft_ReactNative_Cxx_UnitTests</RootNamespace>
+    <RootNamespace>Microsoft.ReactNative</RootNamespace>
     <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.18362.0</WindowsTargetPlatformVersion>
-    <WindowsTargetPlatformMinVersion>10.0.17134.0</WindowsTargetPlatformMinVersion>
+    <WindowsTargetPlatformMinVersion>10.0.15063.0</WindowsTargetPlatformMinVersion>
```
Comment (NikoAri) [inline vnext/Microsoft.ReactNative.Cxx.UnitTests/Microsoft.ReactNative.Cxx.UnitTests.vcxproj]: >15063 [](start = 42, length = 5)

why is there a decrease in versions?

Comment (vmoroz) [reply vnext/Microsoft.ReactNative.Cxx.UnitTests/Microsoft.ReactNative.Cxx.UnitTests.vcxproj]: This is a minimal supported version of WIndows. We must support 15063.0 in order to have C++/WinRT projects load .Net assemblies. In this test project it was not needed, but I want to make sure that tests are compiled with the same settings as other projects.

File: vnext/Microsoft.ReactNative/pch.h (hunk 2 above)
Comment (vmoroz) [reply vnext/Microsoft.ReactNative/pch.h]: It was not intentional - I am going to revert it.
